src/components/RiverConditions_Carousel.vue

```diff
@@ -53,11 +57,6 @@
             },
             getThumbnailUrl(folder, thumbnail) {
                 return 'https://labs.waterdata.usgs.gov/visualizations/river-conditions/' + folder + thumbnail;
-            },
-            handleSlideClick () {
-                this.charts.forEach(chart => {
-                    window.open(chart.drupal_url, "_blank");
```

Comment:
Here, I think you want to give the function the `chart.drupal_url` for the target slide, instead of using `forEach` across all of them